js/view.js

```diff
@@ -5,7 +5,6 @@ projectView.handleMainNav = function() {
     console.log($(this).data('content'));
     $('.tab-content').hide();
     $("#" + $(this).data('content')).show();
```

Comment:
You don't seem to have any elements with class .tab-content or any with the id main-page or about. You need to have those in the HTML to make this bit of JS work.